games-misc/solarus-quest-editor/files/solarus-quest-editor-1.6.5-fix-segfault.patch

```diff
@@ -0,0 +1,22 @@
+Solarus quest editor will segfault on startup with -O2
+
+See: https://gitlab.com/solarus-games/solarus-quest-editor/-/commit/921ac6170a08dcfe601c7a11ee9f7c85386a4f3f
```

Comment:
This is fine, but if it's literally the same commit (or a trivial rebase), instead go to https://gitlab.com/solarus-games/solarus-quest-editor/-/commit/921ac6170a08dcfe601c7a11ee9f7c85386a4f3f.patch, save it as this, then scrub-patch on it (and include the link you did, with the description) so we get some more metadata.